Tools/changelog/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "changelogs.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "axios": "^1.3.4",
+    "js-yaml": "^4.1.0"
+  }
+}
```

Comment:
no npm hell please

Reply:
what

Reply:
> what

why are you using npm for a c# project, cant you just use something else like python that has basic stuff (`requests`) built in

Reply:
I don't know python and I've used JavaScript a lot previously